ResourceGenerator/M365DSCResourceGenerator.psm1

```diff
@@ -1635,8 +1634,7 @@ function Get-Microsoft365DSCModuleCimClass
         $ResourceName
     )
 
-    Import-Module -Name Microsoft365DSC -Force
-    $modulePath = Split-Path -Path (Get-Module -Name Microsoft365DSC).Path
+    $modulePath = "C:\Program Files\WindowsPowerShell\Modules\Microsoft365DSC\Modules\Microsoft365DSC"
```

Comment:
We should not touch this file here. Please undo the changes in the PR. The module can be installed anywhere on the dev's machine and we shouldn't hardcode paths.

Reply:
I thought I removed these changes, let me revert